packages/contracts-bedrock/contracts/dispute/FaultDisputeGame.sol

```diff
@@ -45,6 +45,12 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone {
      */
     Position internal constant ROOT_POSITION = Position.wrap(1);
 
+    /**
+     * @notice The absolute prestate of the instruction trace. This is a constant that is defined
+     *         by the program that is being used to execute the trace, in this case, Cannon.
+     */
```

Comment:
I think we should just not mention cannon at all here so we won't need to change in the future on name change. Keep it abstractly interfaced.

Reply:
Would just leave it at:
```suggestion
    /**
     * @notice The absolute prestate of the instruction trace. This is a constant that is defined
     *         by the program that is being used to execute the trace.
     */
```